wikiman.sh

```diff
@@ -2,13 +2,8 @@
 
 # BSD compatibility: Use GNU find and awk
 
-conf_find='find'
-"$conf_find" --help >/dev/null 2>/dev/null || \
-	conf_find='gfind'
-
-conf_awk='awk'
-"$conf_awk" --help >/dev/null 2>/dev/null || \
-	conf_awk='gawk'
+conf_find=$(command -v find || command -v gfind)
+conf_awk=$(command -v awk || command -v gawk)
```

Comment:
@deuzene Please remove this change so I can merge the commit. Adding the suggested improvement myself would rob you of your contribution.

Reply:
@filiparag I removed those lines in my fork, is that enough to do? Should I do a merge or rebase action?

Reply:
Now it looks the same as it did before your pull request 😕. You even removed the good parts, and left debugging `echo` on [L290](https://github.com/filiparag/wikiman/blob/9e2c8b49b8bd6c1fa3205e7d5d8cf9cedb547a5e/wikiman.sh#L290).

I see that you had a naming conflict of `command` and `$command`. You can rename `$command` variable to `$cmd` to resolve the name conflict.

I am not mad at you or anything, it is your first PR after all. It's just I want to release bugfix version `2.11.4` soon, and this would be a nice addition.

Reply:
@filiparag Thanks for your help. 

I think I put things back on track, take a look please. I don't understand “squash all your commits into one”. :thinking:   Could you help me again?

Reply:
Squashing all commits into one means that you will be only left with one which contains changes from all of them.

Look at this article:
https://www.scraggo.com/how-to-squash-commits/

Reply:
Thank you again. I think it's ok now.

Reply:
Here is what you need to do. First execute:
```bash
git remote add upstream https://github.com/filiparag/wikiman
git fetch upstream
git rebase -i upstream/master
```
Then edit opened text file to match:
```bash
drop a2882b0 Fix SC2181 (which => command -v)
drop a85b2d8 Syncing uppstream
drop 672fabb make more simpler BSD compat. find/awk)
drop c56e31e add /bin path to resolve: "error: unsupported installation path" for Fedora
drop 3e9c37e add /bin path for fedora
drop 8a1089d Fix SC2181 (which => command -v)
drop 4c952d7 make more simpler BSD compat. find/awk)
drop 9e2c8b4 remove the: make more simpler BSD compat. find/awk
reword 8da8fd6 Fix SC2181 (which => command -v)
```
Open `wikiman.sh` and find:
```bash
<<<<<<< HEAD
			case "$(dirname "$(which wikiman)")" in 
=======
			case "$(dirname "$(command -v wikiman)")" in
>>>>>>> 8da8fd6... removed the messing up
```
and replace it with:
```bash
case "$(dirname "$(command -v wikiman)")" in
```
And lastly:
```bash
git add wikiman.sh
git commit -m 'Fix SC2181 (which => command -v)'
git rebase --continue
git push
```
I hope you understand what I did here; if not, feel free to ask what you don't understand.

Reply:
:persevere:  When I finally `git push` I have this issue:

```bash
To https://github.com/deuzene/wikiman
! [rejected]        master -> master (non-fast-forward)
error: failed to push some refs to 'https://github.com/deuzene/wikiman'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details
```

Reply:
Try using `git push --force`. I see that you made a lot of additional commits 😟 Hope you manage to fix it.

Reply:
`--force` worked. I've mistaken and try to undo my changes, I certainly messed up again. I'm very sorry, you are very patient. Again, please guide me, I don't want to damage your work. :fearful:

Reply:
Thank you, Filip.